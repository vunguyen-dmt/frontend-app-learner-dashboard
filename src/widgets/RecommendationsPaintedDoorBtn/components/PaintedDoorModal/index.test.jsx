import { shallow } from 'enzyme';

import ModalView from '.';

jest.mock('../../track', () => ({
  trackPaintedDoorRecommendationHomeSkipBtnClicked: jest.fn(),
  trackPaintedDoorRecommendationHomeInterestBtnClicked: jest.fn(),
}));

describe('ModalView', () => {
  const props = {
    isOpen: true,
    onClose: jest.fn(),
    variation: '',
  };
  describe('snapshot', () => {
    test('should renders default ModalView', () => {
      const wrapper = shallow(<ModalView {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
