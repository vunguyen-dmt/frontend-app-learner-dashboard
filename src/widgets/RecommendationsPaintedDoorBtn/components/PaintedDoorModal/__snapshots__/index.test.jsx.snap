// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalView snapshot should renders default ModalView 1`] = `
<div
  className="containers modal-container"
>
  <ModalDialog
    hasCloseButton={false}
    isBlocking={true}
    isFullscreenScroll={true}
    isOpen={true}
    onClose={[MockFunction]}
    title=""
  >
    <ModalDialog.Header>
      <Component
        className="mt-2"
      >
        Thank you for your interest!
      </Component>
    </ModalDialog.Header>
    <ModalDialog.Body>
      <div>
        <p
          className="mt-2"
        >
          Personalized recommendations feature is not yet available. We are working hard on bringing it to your learner home in the near future.
        </p>
        <p>
          Would you like to be alerted when it becomes available?
        </p>
      </div>
    </ModalDialog.Body>
    <Component>
      <ActionRow>
        <Component
          onClick={[Function]}
          variant="tertiary"
        >
          Skip for now
        </Component>
        <Component
          onClick={[Function]}
          variant="primary"
        >
          Count me in!
        </Component>
      </ActionRow>
    </Component>
  </ModalDialog>
</div>
`;
